### ISSUE TITLE
Unexpected error re. non-empty schemas

### ISSUE THREAD

**Issue by juliahayward** [flyway/flyway#2537]
Reported by David Atkinson:

Confusing error here. The issue is that my HR schema is non empty, but the error also mentions the lack of schema history table, which is not relevant as HR is the second of two schemas, so shouldn't have a schema history table anyway, as this gets created and managed in the first.
Running flyway migrate failed: 
```"C:\Program Files\Red Gate\Redgate Change Control\flyway\flyway.cmd" exited with code 1
Flyway Enterprise Edition (unlimited schemas) 6.0.6 by Redgate licensed to Redgate until 2020-07-01
Database: jdbc:oracle:thin:@//localhost:1521/dev1.red-gate.com (Oracle 18.0)
Successfully validated 18 migrations (execution time 00:00.042s)
ERROR: Found non-empty schema(s) "HR" without schema history table! Use baseline() or set baselineOnMigrate to true to initialize the schema history table.```

**Comment by dvdtknsn** [comment]
My Flyway conf has the following: ]
flyway.schemas=CRM, HR

(so HR will never have a schema history table so suggesting this might be the problem in the error is confusing - the real problem here was the non-empty schema)

**Comment by juliahayward** [comment]
The messages will be clarified in 6.0.8

**Comment by juliahayward** [comment]
We're going to revisit this after we've resolved #1328 as that will have some bearing on the relevant error messages.

**Comment by juliahayward** [comment]
Message clarified in 6.1.3
